### ISSUE TITLE
"mocha": true doesn't seem to work

### ISSUE THREAD

**Issue by jefkoslowski** [AtomLinter/linter-jshint#170]
My .jshintrc file looks like this:

```javascript
{
  "mocha": true,
  "asi": true,
  "esnext": true,
  "noyield": true
}
```

But still I get the error `"describe" is not defined`. The only way to get rid of the errors was to add `/*global describe, it, beforeEach, afterEach */` on top of my mocha test file. I also have tried to use globals, but didn't work either:

```javascript
"globals": {
  "describe": false,
  "it": false,
  "before": false,
  "beforeEach": false,
  "after": false,
  "afterEach": false
}
```

Is this a bug or I am doing something really stupid here?

Atom 1.0.15
linter-jshint 1.2.0
linter 1.6.0
node 0.12.7

**Comment by steelbrain** [comment]
What's the result when you lint from the cli?

**Comment by jefkoslowski** [comment]
So, after all, `linter-js-standard` was the responsible for the error messages. Strangely enough, jshint doesn't complain about this, even on cli.

Thanks @steelbrain :)
